R/spatial_block_cv.R

```diff
@@ -133,14 +152,16 @@ random_block_cv <- function(data, grid_blocks, v) {
   grid_blocks <- sf::st_as_sf(grid_blocks)
   grid_blocks$fold <- sample(rep(seq_len(v), length.out = nrow(grid_blocks)))
 
-  generate_folds_from_blocks(data, grid_blocks, v, n)
+  generate_folds_from_blocks(data, grid_blocks, v, n, radius, buffer)
 }
 
 systematic_block_cv <- function(data,
                                 grid_blocks,
                                 v,
                                 ordering = c("snake", "continuous"),
-                                relevant_only = TRUE) {
+                                relevant_only = TRUE,
+                                radius,
+                                buffer) {
```

Comment:
required args are usually listed before the optional ones

Reply:
In my mind, everything from `v` down is optional, because the user doesn't need to specify those. Is there a reason `radius` and `buffer` should be higher up the list?